pixy/httpapi.go

```diff
@@ -0,0 +1,131 @@
+package pixy
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/gorilla/mux"
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/mailgun/manners"
+)
+
+const (
+	NetworkTCP  = "tcp"
+	NetworkUnix = "unix"
+
+	hContentLength = "Content-Length"
+
+	fTopicName  = "topic"
+	fRoutingKey = "key"
```

Comment:
Thanks for the pointer to the wikipedia article. I still don't know what `f` and `h` mean, though. With sample size n=1, they slightly decrease my understanding of the constants.